win32.mak

```
@@ -1,13 +1,14 @@
 MAKE=make
+HOST_DMD?=dmd
 
 defaulttarget:
 	cd src
-	$(MAKE) -f win32.mak
+	$(HOST_DMD) -run ./src/build.d -v all MODEL=64
```

Wrong model for this file:
```suggestion
	$(HOST_DMD) -run ./src/build.d all MODEL=32
```